docs/source/asdf-schemas.rst

```diff
@@ -0,0 +1,63 @@
+.. _asdf-schemas:
+
+ASDF Schemas
```

Comment:
The manifests aren't schemas. I'm in favor of keeping the old heading:
"ASDF Standard Resources"